.github/workflows/rust.yml

```diff
@@ -134,3 +134,20 @@ jobs:
         cache-on-failure: "true"
     - name: Dry-run publish
       run: cargo publish --dry-run -p ${{ matrix.package }}
+  check-semver:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        package:
+        - generic-ec
+        - generic-ec-core
+        # TODO: this fails because the currently published version doesn't compile
+        # by itself. A fix is present and will be published soon, uncomment this
+        # when it's published
+        # - generic-ec-curves
+        - generic-ec-zkp
+    - uses: actions/checkout@v3
+    - name: Check semver
+      uses: obi1kenobi/cargo-semver-checks-action@v2
+      with:
+        package: ${{ package }}
```

Comment:
I think it should be `${{ matrix.package }}` like above

Reply:
We'll never know because the builds are still stuck..